app/src/main/java/com/techcourse/dao/UserDao.java

```diff
@@ -1,121 +1,53 @@
 package com.techcourse.dao;
 
 import com.techcourse.domain.User;
-import nextstep.jdbc.JdbcTemplate;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import javax.sql.DataSource;
-import java.sql.Connection;
-import java.sql.PreparedStatement;
 import java.sql.ResultSet;
-import java.sql.SQLException;
 import java.util.List;
+import javax.sql.DataSource;
+import nextstep.jdbc.JdbcTemplate;
+import nextstep.jdbc.ObjectMapper;
 
 public class UserDao {
 
-    private static final Logger log = LoggerFactory.getLogger(UserDao.class);
+    private static final ObjectMapper<User> OBJECT_MAPPER = (ResultSet rs) ->
+            new User(rs.getLong("id"),
+                    rs.getString("account"),
+                    rs.getString("password"),
+                    rs.getString("email"));
 
-    private final DataSource dataSource;
+    private JdbcTemplate jdbcTemplate;
```

Comment:
```suggestion
    private final JdbcTemplate jdbcTemplate;
```
인스턴스 변수에 전반적으로 `final` 을 붙이는것 같은데 여기에도 붙이는건 어떨까요?